vyper/compiler/input_bundle.py

```diff
@@ -22,30 +25,33 @@ class CompilerInput:
     # resolved_path is the real path that was resolved to.
     # mainly handy for debugging at this point
     resolved_path: PathLike
+    contents: str
```

Comment:
Can we add a comment to clarify which contents this represents?

Reply:
besides this? https://github.com/vyperlang/vyper/pull/3891/files/63bf65cf497b6e32c52498f6ce3fd5d02db8a92f#diff-03f75f1be3f26d0c0aa9fea620e036473ebe05646dd289a49fbf8f9e034a200eR21

Reply:
Oh wow this looks like it's describing the `source_id` not the `CompilerInput` normally comments are above the line they describe (or docstrings below lol)